fendermint/app/src/options/key.rs

```diff
@@ -21,6 +23,19 @@ pub struct KeyArgs {
     pub command: KeyCommands,
 }
 
+#[derive(Args, Debug)]
+pub struct AddPeer {
+    /// The path to node key file.
+    #[arg(long, short)]
+    pub node_key_file: PathBuf,
+    /// The path to a local file with persistent peers.
+    #[arg(long, short)]
+    pub local_peers_file: PathBuf,
+    /// The path to a local file with persistent peers.
```

Comment:
Wrong docs.

Reply:
If this is a standard CometBFT file, let's mention its default location, so people can find it easier. At the moment I had to check if it's an input or an output.

Reply:
Ah, sorry, this isn't even a file. So let's document the format, of it, mention which port to put in.

Reply:
Fixed